prime-dotnet-webapi/Services/interfaces/IOrganizationClaimService.cs

```diff
@@ -0,0 +1,22 @@
+
+using System.Threading.Tasks;
+using Prime.Models;
+using Prime.ViewModels;
+using Prime.Models.Api;
+
+namespace Prime.Services
+{
+    public interface IOrganizationClaimService
+    {
+        Task<bool> HasClaimAsync(int organizationId);
+
+        /// <summary>
+        /// Returns <c>true</c> if can successfully delete claim, <c>false</c> otherwise.
+        /// </summary>
+        Task<bool> DeleteClaimAsync(int claimId);
+
+        Task<OrganizationClaim> GetOrganizationClaimAsync(int organizationId);
+        Task<Organization> ClaimOrganizationAsync(OrganizationClaimViewModel claimOrganization, Organization organization);
```

Comment:
This method's name is a bit misleading; it sounds like this method might re-assign the Signing Authority. Could you please change it to `CreateOrganizationClaimAsync`?